proparse/src/main/java/org/prorefactor/proparse/Lexer.java

```diff
@@ -986,7 +987,7 @@ void macroDefine(int defType) throws IOException {
     String defText = StringFuncs.stripComments(currText.substring(it));
     defText = defText.trim();
     // Do listing before lowercasing the name
-    prepro.getLstListener().define(textStartLine, textStartCol, macroName.toLowerCase(), defText,
+    prepro.getLstListener().define(textStartLine, textStartCol, macroName.toLowerCase(Locale.ENGLISH), defText,
         defType == ProParserTokenTypes.AMPGLOBALDEFINE ? MacroDef.GLOBAL : MacroDef.SCOPED);
     if (defType == ProParserTokenTypes.AMPGLOBALDEFINE)
       prepro.defGlobal(macroName.toLowerCase(), defText);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define the locale to be used in this String operation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1449)